pkg/asset-manager-utils/test/AaveATokenAssetManager.test.ts

```diff
@@ -491,14 +492,12 @@ describe('Aave Asset manager', function () {
       });
 
       it('transfers the expected number of tokens to the Vault', async () => {
-        const { cash, managed } = await vault.getPoolTokenInfo(poolId, tokens.DAI.address);
-        const poolTVL = cash.add(managed);
-        const targetInvestmentAmount = poolTVL.mul(poolConfig.targetPercentage).div(fp(1));
-        const expectedRebalanceAmount = managed.sub(targetInvestmentAmount);
+        const { poolCash, poolManaged } = await assetManager.getPoolBalances(poolId);
+        const expectedRebalanceAmount = calcRebalanceAmount(poolCash, poolManaged, poolConfig);
 
         await expectBalanceChange(() => assetManager.rebalance(poolId), tokens, [
-          { account: lendingPool.address, changes: { DAI: ['very-near', -expectedRebalanceAmount] } },
-          { account: vault.address, changes: { DAI: ['very-near', expectedRebalanceAmount] } },
+          { account: lendingPool.address, changes: { DAI: ['very-near', expectedRebalanceAmount] } },
+          { account: vault.address, changes: { DAI: ['very-near', -expectedRebalanceAmount] } },
```

Comment:
```suggestion
          { account: vault.address, changes: { DAI: ['very-near', expectedRebalanceAmount,mul(-1)] } },
```

This only works for small BNs.

Reply:
There's a few more instances of this